index.css

```diff
@@ -87,3 +90,11 @@ button {
   box-shadow: var(--box-shadow-lg);
   transition: all 0.3s ease;
 }
+@media (max-width: 640px) {
+  .mobile-hidden {
+    display: none;
+  }
+  .mobile-display {
+    display: block;
+  }
+}
```

Comment:
CSS: Good use of media queries for responsive design. Continue to ensure that the class names are intuitive and maintain consistency across your stylesheets.